### DISCUSSION TITLE
Should we use Supabase or Firebase for backend?

### DISCUSSION THREAD

**Discussion by ykdojo** [ykdojo/defaang#23] [General]
For the backend, should we use Supabase, Firebase or something else?

My idea was to use Supabase because it's more open-sourced and this is an open-source project, but I'm open to ideas/suggestions.

**Comment by VaibhavSingh8** [comment]
I think Supabase will be a good choice as it is obviously open source and it also provides more functionality like rdbms.

**Comment by ykdojo** [comment]
I was just doing some research on Supabase.

This hackathon might be a bonus, if anyone's interested in participating with me: https://supabase.com/blog/launch-week-5-hackathon

**Reply by VaibhavSingh8** [reply]
You mean you want to enter with this project?

**Reply by ykdojo** [reply]
Yes! That's what I meant

**Reply by subhoghoshX** [reply]
That sounds amazing. We should definitely do it.

**Comment by hodeinavarro** [comment]
I'd go with Supabase to follow with the OS idea. 👍🏻

**Comment by thepushkaraj** [comment]
I think using supabase is a good idea, And your point is correct it's more like open-source and it almost provides all the functionalities like Firebase.
